### PR TITLE
SPOI-8848 S3 to HDFS example app

### PR DESCRIPTION
**Pull request by yogidevendra**

### CODE REVIEW

**Comment by yogidevendra** [comment]
@chaithu14 Could you please review this?

**Comment by yogidevendra** [comment]
@chaithu14 Gentle reminder

File: tutorials/s3-to-hdfs-sync/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Amazon S3 to HDFS sync application
```
Comment (chaithu14) [inline tutorials/s3-to-hdfs-sync/README.md]: Please add this app info in tutorials/README.md file.

File: tutorials/s3-to-hdfs-sync/pom.xml [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.apache.apex</groupId>
+      <artifactId>apex-engine</artifactId>
+      <version>${apex.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.janino</groupId>
+      <artifactId>commons-compiler</artifactId>
```
Comment (chaithu14) [inline tutorials/s3-to-hdfs-sync/pom.xml]: Why janino library is needed?

File: tutorials/s3-to-hdfs-sync/src/site/conf/cluster-memory-conf.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
```
Comment (yogidevendra) [inline tutorials/s3-to-hdfs-sync/src/site/conf/cluster-memory-conf.xml]: @ashwinchandrap Could you please explain why are we specifying operator properties in different property file.

Comment (yogidevendra) [inline tutorials/s3-to-hdfs-sync/src/site/conf/cluster-memory-conf.xml]: Single configuration is not suitable for all environments. So, we are trying to capture them separately.

properties.xml would capture mandatory properties for which there cannot be system default.
Other properties which are advanced but can be initialized with meaningful defaults are captured under apconf. These can be used using -apconf from apex cli.

File: tutorials/s3-to-hdfs-sync/pom.xml (hunk 2 above)
Comment (yogidevendra) [inline tutorials/s3-to-hdfs-sync/pom.xml]: removed janino.

File: tutorials/s3-to-hdfs-sync/README.md (hunk 1 above)
Comment (yogidevendra) [inline tutorials/s3-to-hdfs-sync/README.md]: added entry in the tutorials/README.md

**Comment by chaithu14** [comment]
Looks fine to me.

**Pull request merged**